include/deal.II/grid/filtered_iterator.h

```diff
@@ -19,9 +19,11 @@
 
 #include <deal.II/base/config.h>
 #include <deal.II/base/exceptions.h>
+#include <deal.II/base/iterator_range.h>
 #include <deal.II/grid/tria_iterator_base.h>
 
 #include <set>
+#include <tuple>
```

Comment:
This only exists in C++11, so you can't unconditionally `#include` it. You could work around this by using `boost::tuple`, but since you don't use it at all outside the C++11 context, just wrap this in `#ifdef DEAL_II_USE_CXX11`.